README.md

```diff
@@ -102,6 +102,32 @@ After deploying the model endpoint, you can access and test it by using the scri
 python examples/inference/api_server_simple/query_single.py --model_endpoint http://127.0.0.1:8000/gpt2
 ```
 
+## Getting Started With Docker
+This guide will assist you in setting up LLM-on-Ray on With Docker.
+
+#### 1. Build docker with dockerfile
+Software requirement: Ubuntu and Docker
+```bash
+git clone https://github.com/intel/llm-on-ray.git
+cd llm-on-ray
+## You can change any settings here 
+source dev/scripts/start_with_docker.sh
+## Docker flie path is 'dev/docker/Dockerfile.{DOCKER_FILE_NAME}'.
```

Comment:
We should move all CI-only Dockerfile in dev/docker/* and move them to a separate directory (e.g. dev/docker/ci). Leaving this directory for scripts can be used for both users and CI.
Need a description in doc that each dockerfile is for what purpose.